public/app/assets/stylesheets/custom.scss

```diff
@@ -73,4 +73,7 @@ ul a {
 }
 .card-text-bold {
     font-weight: 600;
+}
+a {
```

Comment:
We can add `color:#181A7B !important;` instead of commenting out the color in `theme.css`.

Reply:
#content a {
    color:#181A7B !important;
}